custom_components/huesyncbox/config_flow.py

```diff
@@ -20,6 +20,7 @@
     CONF_UNIQUE_ID,
 )
 from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.service_info.zeroconf import ZeroconfServiceInfo
```

Comment:
I guess the old import for zeroconf is unused now. 
If so it should be removed.

Reply:
Done :)